### ISSUE TITLE
"Could not deserialize as static object"

### ISSUE THREAD

**Issue by NagRock** [swc-project/swc#7367]
### Describe the bug

I'm getting random build fails on Jenkins CI.

When I had `maxWorkers` set to 2-4 it was usually just one error like:

```
A jest worker process (pid=1989) was terminated by another process: signal=SIGBUS, exitCode=null. Operating system logs may contain more information on why this occurred.
```

When I've increased `maxWorkers` to `50%` on AWS with `40 CPUs` I've received a lot of such errors and also a lot of:

```
[2023-05-08T17:47:21.816Z] Could not deserialize as static object: Feature static load is not compiled in is not yet supported
[2023-05-08T17:47:22.236Z] Could not deserialize as static object: Feature static load is not compiled in is not yet supported
[2023-05-08T17:47:22.716Z] Could not deserialize as static object: Feature static load is not compiled in is not yet supported
[2023-05-08T17:47:22.716Z] Could not deserialize as static object: Feature static load is not compiled in is not yet supported
[2023-05-08T17:47:23.616Z] FAIL src/app/hooks/use-run.spec.ts
[2023-05-08T17:47:23.616Z]   ● Test suite failed to run
[2023-05-08T17:47:23.616Z] 
[2023-05-08T17:47:23.616Z]     A jest worker process (pid=1955) was terminated by another process: signal=SIGBUS, exitCode=null. Operating system logs may contain more information on why this occurred.
[2023-05-08T17:47:23.616Z] 
[2023-05-08T17:47:23.616Z]       at ChildProcessWorker._onExit (../../node_modules/jest-worker/build/workers/ChildProcessWorker.js:370:23)
[2023-05-08T17:47:23.616Z] 
[2023-05-08T17:47:24.017Z] PASS src/runtime/loader.spec.ts (11.395 s)
[2023-05-08T17:47:24.515Z] FAIL src/use-applications.spec.ts
[2023-05-08T17:47:24.515Z]   ● Test suite failed to run
[2023-05-08T17:47:24.515Z] 
[2023-05-08T17:47:24.515Z]     A jest worker process (pid=1989) was terminated by another process: signal=SIGBUS, exitCode=null. Operating system logs may contain more information on why this occurred.
[2023-05-08T17:47:24.515Z] 
[2023-05-08T17:47:24.516Z]       at ChildProcessWorker._onExit (../../node_modules/jest-worker/build/workers/ChildProcessWorker.js:370:23)
.
.
.
.
.
```

### Found place with error message:

**Then I've searched `node_modules` to find `Could not deserialize as static object` and the only result is: `swc.win32-x64-msvc.node`** 

### Version

"@swc/core": "1.3.56",     "@swc/jest": "0.2.26",

**Comment by kwonoj** [comment]
There is no way to investigate this further without correct repro and additional details.

**Comment by NagRock** [comment]
Yes, I understand. But it also hard to provide some details when it appears randomly and only one place with this error message comes from swc compiled file.

I will update thread if I'll get some more info.

**Comment by kwonoj** [comment]
> swc compiled file.

I did quick search in the repo but cannot find corresponding errors. Where does it comes from?

**Comment by NagRock** [comment]
I found it in `swc.win32-x64-msvc.node` by searching for any text occurrence in my `node_modules` but this file got ~51MB 

Failing setup:
```
    "@swc/core": "1.3.49",
    "@swc/jest": "0.2.26",
    "jest_workaround": "0.75.4",
```

Last working setup:
```
    "@swc/core": "1.3.36",
    "@swc/jest": "0.2.24",
    "jest_workaround": "0.1.14",
```

I've tried other versions but all between `1.3.36 - 1.3.49` - have different errors.

**Comment by kwonoj** [comment]
At this point honestly I'm not very much convinced this is caused by swc itself, more sounds like plugin jest_workaround needs some dependency updates.

**Comment by wokkaflokka** [comment]
I can also confirm working on a project with similar characteristics and dependencies observing similar behaviors in CI/CD.

We are primarily observing a different failure -- `Bus error (core dumped)` -- but I noticed a log with the text `Could not deserialize as static object: Feature static load is not compiled in is not yet supported` which led me to this thread.

The project I am investigating uses all three of the dependencies noted here.

```
    "@swc/core": "1.3.57",
    "@swc/jest": "0.2.26",
     "jest": "29.5.0",
     "jest_workaround": "0.76.0",
```

I checked `jest_workaround` and the project dependencies appear to be as follows:

```
    "devDependencies": {
        "@swc/core": "^1.3.57",
        "@swc/jest": "^0.2.26",
        "@types/jest": "^29.5.1",
        "jest": "^29.5.0"
    },
    "peerDependencies": {
        "@swc/core": "^1.3.57",
        "@swc/jest": "^0.2.26"
    }
```

Per the comments from `@kwonoj`, I recognize that my comment isn't particularly helpful as it is not providing repro or much in the way of details. I am not sure that the issue I am investigating relates to SWC. But I didn't want to comment as I reviewed this thread in case this provides anything of value.
